fix_includes_test.py

```diff
@@ -3713,6 +3717,27 @@ def testAddForwardDeclareInNamespaceWithKeepIwyuNamespaceFormat(self):
     self.RegisterFileContents({'add_fwd_declare_keep_iwyu_namespace': infile})
     self.ProcessAndTest(iwyu_output, expected_num_modified_files=1)
 
+  def testBasePath(self):
+    self.flags.base_path = "basepath/"
```

Comment:
I think this test would be clearer if you mimicked the build-dir/src-dir tension and with absolute paths, something like:

```
self.flags.base_path = "/project/build"
iwyu_output = """\
../src/source.cc should add these lines:
...
"""
...
self.RegisterFileContents({'/project/src/source.cc': infile})
```

And a doc comment for the test would also help.